src/shaders/icosahedron-vert.glsl

```diff
@@ -0,0 +1,69 @@
+varying vec3 norm;
+varying vec2 vUv;
+uniform float time;
+uniform float noise_scale;
+
+
+float noise_1(float x, float y, float z) {
+  return cos(z * sin(dot(vec2(x, y), vec2(12.9898, 78.233))) * 43758.5453);
+}
+
+float noise_2(float x, float y, float z) {
```

Comment:
I know you didn't use this and `noise_3`, but just so you know, this isn't going to be pseudorandom